help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -156,31 +156,32 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
 ![Platform UI screenshot example for Source mapping.](../../assets/catalog/crm/microsoft-dynamics-365/source-mapping.png)
 
 1. In the **[!UICONTROL Select target field]** window, select the type of target field that you want to map your source field to.
     * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
-    ![Platform UI screenshot showing Target mapping for contactId.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-contactid.png)
+    ![Platform UI screenshot showing Target mapping for contactid.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-contactid.png)
 
     * Add the following mapping between your XDM profile schema and your [!DNL Dynamics 365] instance:
         |XDM Profile Schema|[!DNL Dynamics 365] Instance| Mandatory|
         |---|---|---|
-        |`contactId`|`contactId`| Yes |
+        |`contactid`|`contactid`| Yes |
 
     * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the **[!UICONTROL Attribute name]** field. Refer to [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1#entity-properties) for a comprehensive list of supported attributes.
-    ![Platform UI screenshot showing Target mapping for LastName.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-lastname.png)
+    ![Platform UI screenshot showing Target mapping for email.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-email.png)
 
         >[!IMPORTANT]
         >
-        >If you have a date or timestamp source field which is mapped to a [!DNL Dynamics 365] [date or timestamp](https://docs.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/timestampdatemapping?view=dataverse-latest) target field, ensure that the mapped value being is not empty. If the value passed is empty you will encounter a *`Bad request reported while pushing events to the destination. Please contact the administrator and try again.`* error message and the data will not be updated. This is a [!DNL Dynamics 365] limitation.
+        > * Target field names should be in `lowercase`.
+        > * Additionally, if you have a date or timestamp source field which is mapped to a [!DNL Dynamics 365] [date or timestamp](https://docs.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/timestampdatemapping?view=dataverse-latest) target field, ensure that the mapped value being is not empty. If the value passed is empty you will encounter a *`Bad request reported while pushing events to the destination. Please contact the administrator and try again.`* error message and the data will not be updated. This is a [!DNL Dynamics 365] limitation.
```

Comment:
minor: 

"ensure that the mapped value being is not empty." -> no need for `being` here